README.md

```diff
@@ -1,3 +1,59 @@
 # Data Models
 
 Data models and vocabularies in the biomedical space.
+
+## Persistent CSV Format
+
+Data model descriptions are stored persistently in this repository in csv format for portability and human readability. Each data model has its own directory with versions of the model in subdirectories. Each version has at least `definitions` and `schema` directories and, optionally, a `constraints` directory and `indexes.csv` and `references.csv` files.
+
+The `definitions` directory holds basic information about the data model that would be of primary interest to a data user. There is a `tables.csv` file, which lists table names and descriptions, as well as a csv file for each table, which lists field names and descriptions, whether the field is "required" (a governance- not data-level attribute), and optionally a table/field combination to which the field refers (typically manifested as a foreign key relationship).
```

Comment:
I think it is more clear to state "schema attribute" rather than "data-level attribute", but I will defer to @murphyke since he is a good editor.